### PR TITLE
fix: fixes typo on tooltip

### PR DESCRIPTION
**Pull request by figueiredo-lucas**
## Description
Fixes typo in variable that would prevent the component from rendering

## Context
There was a typo in tooltip.marko when having more than one series that would give several errors on log and prevent the tooltip to show up

## References
https://github.com/eBay/ebayui-core/issues/1866

## Screenshots
Can be found in the issue

### CODE REVIEW

**Review by LuLaValva — APPROVED** [review]
Great catch, thanks for the PR! Looks good on my end, but I'm not sure if we should do a patch release or put this out with `16.0.0` so we should wait for @agliga's approval before merging.

**Comment by agliga** [comment]
Lets target 12.0.0 with this for now.

**Review by ianmcburnie — APPROVED** [review]

**Review by agliga — APPROVED** [review]

**Pull request merged**
